satellite/satellitedb/overlaycache.go

```diff
@@ -1421,20 +1421,20 @@ func (cache *overlaycache) UpdateCheckIn(ctx context.Context, node overlay.NodeC
 				free_disk=$9,
 				major=$18, minor=$19, patch=$20, hash=$21, timestamp=$22, release=$23,
 				total_uptime_count=nodes.total_uptime_count+1,
-				uptime_reputation_alpha=$16::numeric*nodes.uptime_reputation_alpha + $17::numeric*$10::bool::int,
-				uptime_reputation_beta=$16::numeric*nodes.uptime_reputation_beta + $17::numeric*(NOT $10)::bool::int,
+				uptime_reputation_alpha=$16::float*nodes.uptime_reputation_alpha + $17::float*$10::bool::int::float,
```

Comment:
I was worried about all these changes from `numeric` to `float`, but as far as I can tell, these are all _more_ correct this way than they were before. @JessicaGreben do you remember why I suggested `numeric` in the first place? Was I just guessing the types in the table and never followed up?